types/jest/index.d.ts

```diff
@@ -801,6 +802,164 @@ declare namespace jest {
 
     type SnapshotUpdateState = 'all' | 'new' | 'none';
 
+    interface DefaultOptions {
+        automock: boolean;
+        bail: boolean;
+        browser: boolean;
+        cache: boolean;
+        cacheDirectory: Path;
+        changedFilesWithAncestor: boolean;
+        clearMocks: boolean;
+        collectCoverage: boolean;
+        collectCoverageFrom: Maybe<string[]>;
+        coverageDirectory: Maybe<string>;
+        coveragePathIgnorePatterns: string[];
+        coverageReporters: string[];
+        coverageThreshold: Maybe<{global: {[key: string]: number}}>;
+        errorOnDeprecated: boolean;
```

Comment:
This one is added in v23.1.0, we should either bump the version at Line 1 to `v23.1` or just remove it.